doc/embedded/crosscompile_example.md

```diff
@@ -0,0 +1,132 @@
+## Crosscompile mlpack example for an embedded hardware
+
+In this article, we explore how to add mlpack to a CMake project that cross-compiles
+code to embedded hardware.  See also these related other guides, which may be useful
+to read before this one:
```

Comment:
```suggestion
to read before this one:

```

(I'm paranoid about how Markdown renders, sometimes without the space it might not render as a list.)